features/partners.feature

```diff
@@ -191,8 +197,7 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
 
 
   @proxy
-  Rule: Als er een actueel huwelijk of geregistreerd partnerschap is wordt alleen dit actueel huwelijk of geregistreerd partnerschap wordt teruggegeven,
-        als er geen actueel huwelijk of geregistreerd partnerschap is, maar wel een ontbonden huwelijk of geregristreerd partnerschap dan wordt alleen het meest recente ontbonden huwelijk of geregistreerd partnerschap geleverd.
+  Rule: Alleen een actueel huwelijk of geregistreerd partnerschap wordt teruggegeven
```

Comment:
ik denk dat dit per ongeluk is gebeurd, maar je hebt zelf in P.R. #1051 regels 194 en 195 toegevoegd, zodat ook achter de proxy de laatst ontbonden partner wordt doorgegeven wanneer er geen actuele partner is. Het is denk ik niet de bedoeling dat we die wijziging hier terugdraaien

Reply:
zie ook scenario "Huwelijk/partnerschap is ontbonden <omschrijving>" (regel 256)

Reply:
Aangepast